components/FeatureHighlight/src/private/MDCFeatureHighlightView+Private.m

```diff
@@ -215,19 +215,19 @@ - (void)updateBodyFont {
   if (!_bodyFont) {
     _bodyFont = [MDCFeatureHighlightView defaultBodyFont];
   }
+  UIFont *bodyFont = _bodyFont;
   if (_mdc_adjustsFontForContentSizeCategory) {
-    if (_bodyFont.mdc_scalingCurve && !_mdc_legacyFontScaling) {
+    if (bodyFont.mdc_scalingCurve && !_mdc_legacyFontScaling) {
       // The font has an associated curve (M2+)
-      _bodyLabel.font = [_bodyFont mdc_scaledFontForCurrentSizeCategory];
-    } else {
+      bodyFont = [bodyFont mdc_scaledFontForTraitEnvironment:self];
```

Comment:
Do we need to set the font twice here [due to the way `UILabel` attempts to optimize `UIFont` setting](https://github.com/material-components/material-components-ios/issues/7424)?